### PR TITLE
[WIP] Login implementation

### PR DESCRIPTION
**Pull request by vpetrusenko**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,14 +1,12 @@
 import React, { Component } from 'react';
-import Button from '@material-ui/core/Button';
+import Page1 from './components/Page1';
```
Comment (sasha240100) [inline src/App.tsx]: `Page1` - Maybe better a to have a proper page that describes better its purpose?

File: src/components/LoginForm.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  },
+};
+
+function LoginForm(props: any) {
+  const { classes } = props;
+  return (
+    <Card className={classes.card}>
+      <CardMedia
+        className={classes.media}
+        image={logo}
```
Comment (sasha240100) [inline src/components/LoginForm.tsx]: Alternative way could be inlining with require:
`image={require('../logo.png')}` 
I consider it as "a better way" for assets.

File: src/components/Page1.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import React, {Component} from 'react'
+import Grid from '@material-ui/core/Grid'
+import LoginForm from '../components/LoginForm'
+
+
```
Comment (sasha240100) [inline src/components/Page1.tsx]: remove 1 empty line

File: src/components/TextField.tsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
```
Comment (sasha240100) [inline src/components/TextField.tsx]: PropTypes imported but not used.

File: src/components/Login.tsx [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+import Grid from '@material-ui/core/Grid'
+import Card from '@material-ui/core/Card';
+import PropTypes from 'prop-types';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+import TextField from '@material-ui/core/TextField';
```
Comment (akhlopiachyi) [inline src/components/Login.tsx]: You can use easier way to import components..
import {
  Grid,
  Card,
  CardActionArea,
  CardActions,
  CardContent,
  CardMedia,
  Button,
  Typography,
  TextField,
} from '@material-ui/core';

File: src/components/Login.tsx [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+      left: 10,
+      height: 48,
+      width: 280,
+      backgroundColor: '#3598D5',
+      alignItems: 'center',
+    }
+ });
+
+class Login extends React.Component<WithStyles<typeof styles>>  {
+
```
Comment (akhlopiachyi) [inline src/components/Login.tsx]: remove empty line

**Comment by akhlopiachyi** [comment]
Remove Vector.png if you don't use it

**Pull request closed without merging**